src/utils/index.ts

```diff
@@ -2,16 +2,17 @@ import { BigNumber, ContractInterface, ContractTransaction, ethers, Signer, util
 import { Provider } from '@ethersproject/abstract-provider';
 import ERC20ABI from 'abis/ERC20.json';
 import TokenGeyserABI from 'abis/TokenGeyser.json';
+import { BigNumber as BNJS } from 'bignumber.js';
 
 export function commifyString(number: string): string {
   const parts = number.split('.');
   parts[0] = parts[0].replace(/\B(?=(\d{3})+(?!\d))/g, ',');
   return parts.join('.');
 }
 
-export function formatRelays(relays: string | BigInt): string {
-  const relaysByMillion = BigInt(relays) / 1000000n;
-  return relaysByMillion.toString();
+export function formatRelays(relays: string | BNJS): string {
+  const relaysByMillion = new BNJS(relays).div(new BNJS(1000000));
+  return relaysByMillion.toFixed(2);
```

Comment:
Why is this fixed to `2`? Just to set the display to 2 decimals? Or any funky token math?

Reply:
Nothing special, maxRelays are calculated depending on how much wPOKT is staked. That's why it's cast to BigNumber - but this doesn't actually represent any token.

Relays are at most 15 million, since we are using 'M' to show numbers as `10 M` then we decided to show only 2 decimals.

Samples:

`2.00 M relays`
`10.50 M relays`
`5.47 M relays`